### PR TITLE
Notify user about successful request of the scaling action.

### PR DESCRIPTION
**Pull request by konstan**
CLI prints message to say that the scaling action request was successful.

Connected to #153

### CODE REVIEW

**Comment by konstan** [comment]
Connected to #153

File: client/src/main/python/ss-scale-disk.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def _validate_and_set_scale_options(self):
 
     def doWork(self):
         super(MainProgram, self).doWork()
+        action = self.options.attach_gb and 'attach' or 'detach'
+        print('Requested %sment of extra disk on %s' % (action, self.node_name))
```
Comment (loomis) [inline client/src/main/python/ss-scale-disk.py]: maybe 'successful %sment..' instead?  'Requested' makes it sound like it hasn't happened yet.

File: client/src/main/python/ss-scale-resize.py [hunk 2]
```diff
@@ -55,6 +55,8 @@ def _validate_and_set_scale_options(self):
 
     def doWork(self):
         super(MainProgram, self).doWork()
+        print('Requested resizing of %s' % self.node_name)
```
Comment (loomis) [inline client/src/main/python/ss-scale-resize.py]: Same comment as above.

File: client/src/main/python/ss-scale-disk.py (hunk 1 above)
Comment (konstan) [inline client/src/main/python/ss-scale-disk.py]: After checking with @loomis. The call is asynchronous, os it's OK to leave this wording.

File: client/src/main/python/ss-scale-resize.py (hunk 2 above)
Comment (konstan) [inline client/src/main/python/ss-scale-resize.py]: After checking with @loomis. The call is asynchronous, os it's OK to leave this wording.

**Pull request merged**